src/report.sh

```diff
@@ -429,9 +457,9 @@ function parse_report_options()
     esac
   done
 
-  if [[ -n "${options_values['STATISTICS']}" ]] && [[ -n "${options_values['OUTPUT']}" ]]; then
-    options_values['ERROR']='--output cannot be used with --statistics.'
-    return 22 # EINVAL
+  if [[ -z "${options_values['STATISTICS']}" ]] && [[ -z "${options_values['POMODORO']}" ]] && [[ -z "${options_values['ALL']}" ]]; then
```

Comment:
All these tests can be inside a single `[[ ... ]]` like this:
```suggestion
  if [[ -z "${options_values['STATISTICS']}" && -z "${options_values['POMODORO']}" && -z "${options_values['ALL']}" ]]; then
```